pyprocessing/attributes.py

```diff
@@ -0,0 +1,30 @@
+from pyprocessing import PyProcessing
+
+
+def stroke_cap(cap):
+    pp = PyProcessing()
+    if cap == 1:
+        pp.namespace['cap'] = 'butt'
+    elif cap == 2:
+        pp.namespace['cap'] = 'round'
+    elif cap == 4:
+        pp.namespace['cap'] = 'projecting'
+    else:
+        pp.namespace['cap'] = 'butt'
+
+
+def stroke_weight(weight):
+    pp = PyProcessing()
+    pp.namespace['stroke_thickness'] = weight
+
+
+def stroke_join(join):
+    pp = PyProcessing()
+    if join == 2:
+        pp.namespace['join'] = 'round'
```

Comment:
as I said in the prrevious review, use more explicit attribute names for namespace attributes, as they can collide with other attributes down the line. In this instance, `stroke_join_method` would be appropriate in my opinion. Same goes for cap

Reply:
I've renamed the namespace attributes to `stroke_join_attribute`. You can't rename the method, as it's the function the user calls.

Reply:
I wasn't talking about the function name but just the namespace's attributes. It better to be a bit descriptive for those to avoid trouble later (like name conflicts, or non descriptive names)